README.md

```diff
@@ -16,20 +16,30 @@ following new functionality:
 
 ## Getting Started and Documentation
 
-Follow these steps to get started with `provider-argocd`.
+Follow the [official docs](https://crossplane.io/docs/master/getting-started/install-configure.html#install-crossplane) to install crossplane, then these steps to get started with `provider-argocd`.
 
 ### Optional: Start a Local ArgoCD Server
+```bash
+kind create cluster
```

Comment:
Do the rest of the docs assume `kind`?

Reply:
crossplane upstream docs use `kind` as well, but I can't think of a case where it matters. Do you have something in mind?

Reply:
I was just thinking add a line saying "you need a local cluster," but didn't specify what cluster.  :-)